recipes/netgen/build.sh

```diff
@@ -0,0 +1,25 @@
+echo "-------------------CUSTOM VARIABLES-------------"
+export NETGENDIR=$PREFIX/lib/netgen
+echo "set NETGENDIR to: $NETGENDIR"
+
+export PYTHONPATH=$SP_DIR
```

Comment:
Is this actually needed? It probably shouldn't be, though maybe netgen has a bad CMakeLists for some reason.

Reply:
I don't know if it work without this. And yes netgens CMakeLists could be improved. But I am happy that it works at all.